core/src/test/java/io/grpc/internal/ServiceConfigInterceptorTest.java

```diff
@@ -312,10 +312,7 @@ public void handleUpdate_failsOnMissingServiceName() {
     thrown.expect(IllegalArgumentException.class);
     thrown.expectMessage("missing service");
 
-    ManagedChannelServiceConfig parsedServiceConfig =
-        createManagedChannelServiceConfig(serviceConfig);
-
-    interceptor.handleUpdate(parsedServiceConfig);
+    createManagedChannelServiceConfig(serviceConfig);
```

Comment:
rename test since it is no longer testing handleUpdate. there are few more tests need to renamed too. those should be moved to `ManagedChannelServiceConfigTest` because it is only testing ManagedChannelServiceConfig.

Reply:
Good catch. Done.

Reply:
Oh, you want to move them to another class. Sure, although some of the tests are out of the scope of current PR.

Reply:
yes, moving makes more sense (only tests you just renamed). should be fairly simple to move, if not i am fine as it is.